src/module.c

```diff
@@ -7672,6 +7677,18 @@ void moduleInitModulesSystem(void) {
     pthread_mutex_lock(&moduleGIL);
 }
 
+/* Close the pipes opened by moduleInitModulesSystem(). */
+void closeModuleBlockedPipe(void) {
```

Comment:
so now that we removed the call in prepareForShutdown, we can remove this function

Reply:
I thought we are gonna keep this function for future uses?

Reply:
By the way, just being curious, isn't really there anywhere in the project where you need to close these pipes in the current redis server?

Reply:
AFAIU what we wanted to keep for future use is the close on exec flag even for parts of redis that never do exec. (cluster and modules).
specifically because it doesn't add much code (one flag in an existing call, or one new line)
but i don't see a reason to keep dead code for future use (specifically because it's so easy to write).

and yes, redis never closes these pipes.

Reply:
Understood.